pkgs/development/interpreters/lua-5/tests/assert.sh

```diff
@@ -3,19 +3,19 @@
 # Example:
 #     fail "It should have been but it wasn't to be"
 function fail() {
-    echo "$1"
+    echo -e "$1"
```

Comment:
`echo -e` evaluates backslash escape sequences, did you mean to do `>&2` to print to stderr?

Reply:
nope, I wanted to echo the newlines. I am not sure what's best actually, do several fail calls or keep the newline ?  anyway this patch made actually harder to look at teh differences because $1  and $2 now dont start on same line. 
What I would like best is to show inline diffs. I know git diff can be used outside repos, not sure about enabling inline diffs, wether it's worth or not.